to-do-list.js

```diff
@@ -1,18 +1,29 @@
 const input = document.getElementById('text-field');
 
-const createListItem = function() {
-  const listItem = document.createElement('li');
+const createTask = function() {
+  const liCreate = document.createElement('li');
+  const spanCreate = document.createElement('span');
+  const classCheck = function(event) {
```

Comment:
Extract it from the function to another in global scope, it will be more readable